pages-css/login-page-css/page-style.css

```diff
@@ -0,0 +1,69 @@
+:root {
+    --color-light-gray: #f2f2f2;
+    --color-dark-orange: #ff8849;
+    --color-light-orange: #ffc591;
+    --color-gray: #4d4d4d;
+    --color-dark: black;
+    --color-light-orange: #ffeed9;
+}
+
+.section-1 {
+    padding-bottom: 1rem;
+    background: var(--color-light-orange);
```

Comment:
try not use bg-color for section UX isn't that appealing

Reply:
ok